content/en/_index.html

```diff
@@ -1,81 +1,55 @@
 +++
-title = "Goldydocs"
-linkTitle = "Goldydocs"
+title = "COAsT"
+linkTitle = "COAsT"
 
 +++
 
-{{< blocks/cover title="Welcome to Goldydocs: A Docsy Example Project!" image_anchor="top" height="full" color="orange" >}}
+{{< blocks/cover title="Welcome to the doucmention: A Docsy site for COAsT" image_anchor="top" height="full" color="orange" >}}
```

Comment:
Documentation has a spelling mistake here. As it's the first page I thought I'd point it out